Sources/App/Core/PackageCollection+generate.swift

```diff
@@ -295,3 +299,13 @@ private extension PackageCollection.Platform {
         }
     }
 }
+
+
+extension PackageCollectionModel.V1.Signer {
+    static var spi: Self {
+        .init(type: "ADP",
+              commonName: "Swift Package Index",
+              organizationalUnitName: "Swift Package Index",
+              organizationName: "Swift Package Index")
```

Comment:
@yim-lee Hi Yim, I'm adapting our package collection handling to the changes in the release/5.9 branch of SwiftPM and I was wondering what the recommended values are for these fields, in particular `type`. I saw mention of `ADP` in the doc string but found no other list of options or anything. I'm assuming `ADP` is short for `Apple Developer Portal` and would be what we should be using for our dev portal issued cert?

Reply:
Hi Sven, the `signer` field is optional and the only supported signer type is `ADP` (Apple Developer Program), so only fill this in if you are signing with an ADP cert (which you are). `commonName` etc. should match the corresponding subject fields in the certificate. Please refer to the [docs](https://github.com/apple/swift-package-manager/blob/main/Sources/PackageCollectionsModel/Formats/v1.md) and let me know if any of it is unclear.